### ISSUE TITLE
Dnsmasq not start after reboot or container restart

### ISSUE THREAD

**Issue by serhiicherepanov** [jderusse/docker-dns-gen#14]
Hi, I run container with flag `--restart always` but after reboot the dnsmasq not found in process list. My host machine switched to slave dns server and lose all docker hosts.

To reproduce this issue just do:

```
docker stop dns-gen
docker start dns-gen
docker top dns-gen #you can see only sh and docker-gen process
```

I think it's require more flexible process management, something like supervisord but more lightweight, I found alternatives wtiren on Go: https://github.com/codeskyblue/gosuv and http://governator.io/

**Comment by serhiicherepanov** [comment]
@jderusse what you think about this?

**Comment by jderusse** [comment]
fixed by #28
